### PR TITLE
Add more comprehensive cilium examples

### PR DESCRIPTION
**Pull request by CiiDyR**
Currently the guide is hard to follow in regards of getting cilium to run properly.
This aims to add the required minimum configmap settings to get cilium to provide a network for the cluster.

### CODE REVIEW

**Review by LKaemmerling — APPROVED** [review]

**Comment by LKaemmerling** [comment]
Good catch! Thank you

**Comment by CiiDyR** [comment]
Whoops, seems like I made a mistake. Blacklisting should be `false`, not `true`. Should I make a new PR?

**Comment by LKaemmerling** [comment]
Yep, would be cool. I overlooked it too.
